src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/AggregateProcessPPIO.java

```diff
@@ -33,6 +37,10 @@ protected MapperWrapper wrapMapper(MapperWrapper next) {
             };
         };
         xstream.allowTypes(new Class[] { AggregateProcess.Results.class });
+        xstream.omitField(AggregateProcess.Results.class, "aggregateAttribute");
```

Comment:
Yes it is worth adding documentation, @bmmpxf is our usual contact point for docs.

Your examples are great, I think we could make a wps-cookbook of examples somewhere here:

* http://docs.geoserver.org/latest/en/user/services/wps/index.html

I would put it after the section on using the request builder, so you can use the request builder when making the examples/requests.